src/Microsoft.DotNet.Build.CloudTestTasks/WriteItemsToJson.cs

```diff
@@ -75,7 +76,25 @@ public override bool Execute()
                             }
                             else
                             {
-                                jsonWriter.WriteValue(mdValue);
+                                if (mdValue.StartsWith("{") && mdValue.EndsWith("}"))
+                                {
+                                    // If it's a JObject, parse it and use that to write...
+                                    try
+                                    {
+                                        JObject jsonEntry = JObject.Parse(mdValue);
+                                        jsonEntry.WriteTo(jsonWriter);
+                                    }
+                                    // Leave it in... it's probably bad by here but writing it aids debugging.
+                                    catch
+                                    {
+                                        jsonWriter.WriteValue(mdValue);
+                                    }
+                                }
+                                // Plain value
```

Comment:
When is this ("Plain value") an acceptable state to be in for a "WriteItemsToJson" task?  I understand the "catch" above aiding with debugging, but it seems like the if and the else here could be removed and just leave the try-catch.